lib/perl/Genome/Model/Tools/Analysis/DumpIgvXmlBasic.pm

```diff
@@ -0,0 +1,97 @@
+package Genome::Model::Tools::Analysis::DumpIgvXmlBasic;
+
+use strict;
+use warnings;
+
+use Genome;
+use Command;
+use IO::File;
+use Cwd qw(abs_path);
+use URI;
+use XML::Writer;
+
+class Genome::Model::Tools::Analysis::DumpIgvXmlBasic {
+    is => 'Command',
+    has => [
+        resource_files => {
+            type => 'HASH',
```

Comment:
I honestly don't know. I think these igv tools are primarily called inside of Perl code, not on the command line. @tmooney do you know?

What's the alternative here if we can't use HASH?